NEWS.md

```diff
@@ -1,80 +1,116 @@
 # ggplot2 3.1.0.9000
 
-* `geom_rug()` gains a `length` option to allow for changing the length of the rug lines. (@daniel-wells, #3109)
+This is a minor release with a main emphasis on internal changes to make ggplot2 
+faster and more consistent. The few breaking changes there are will only affect
+the aesthetics of the plot (but not break code) or, potentially, extension 
+developers if they have relied on internals that have been changed.
 
-* `coord_sf()` graticule lines are now drawn in the same thickness as 
-  panel grid lines in `coord_cartesian()`, and seting panel grid
-  lines to `element_blank()` now also works in `coord_sf()` 
+## Breaking Changes
+
+* Default labels are now generated more consistently; e.g., symbols no longer
+  get backticks, and long expressions are abbreviated with `...`
+  (@yutannihilation, #2981).
+
+* `GeomRibbon` and `GeomArea` now sort the data along the x-axis in the 
```

Comment:
Why is this a breaking change? Given that no revdeps broke, I don't actually understand why we even have this section.

Reply:
It could potentially break someones code if they had subclassed one of these (but not put it in a package)... I'm fine with removing the whole section, though

Reply:
Looking at it from the perspective of a ggplot2 user with code generating hundreds of carefully crafted figures, I'd rather know if there are code changes that are likely to cause perceptible differences to my figures. Not sure if those changes are breaking, in the strict definition of the term, but they do deserve to be called out in a separate section I think.

Reply:
I've currently removed the section completely... I don't care much either way, so the one with the strongest feelings about it gets to decide

Reply:
I think the problem is getting the granularity right - if you describe things that only subtly affect a few plots in minor ways, they will swamp out the actually important stuff.

But I'd be ok renaming this section to "Visual changes" or similar — it's just that "breaking changes" has a specific meaning in our process.

Reply:
I read through the entire news file as is top to bottom and I think it's fine as it currently stands, without an explicit "breaking changes" section.